README.md

```diff
@@ -325,12 +325,17 @@ spec:
       managementState: Managed
     modelmeshserving:
       managementState: Managed
+    modelregistry:
+      managementState: Managed
+      registriesNamespace: "odh-model-registries"
     ray:
       managementState: Managed
     trainingoperator:
       managementState: Managed
     workbenches:
       managementState: Managed
+    modelregistry:
+      managementState: Managed
```

Comment:
```suggestion
      managementState: Removed
```

Reply:
this hunk probably came with some automatic merge, 2 modelregistry entries